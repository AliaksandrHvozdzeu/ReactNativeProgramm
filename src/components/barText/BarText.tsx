import React from 'react';
import {Text} from 'react-native';
import {styles} from './styles';

type barTextProps = {
  text: string;
};

const BarText = ({text}: barTextProps) => {
  return <Text style={styles.name}>{text}</Text>;
};

export default BarText;
