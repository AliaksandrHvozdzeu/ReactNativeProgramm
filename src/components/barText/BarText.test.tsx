import 'react-native';
import React from 'react';
import {test, describe, expect} from '@jest/globals';
import BarText from './BarText';
import {render} from '@testing-library/react-native';

describe('BarText tests', () => {
  test('Renders the Footer correctly', () => {
    const tree = render(<BarText text={text} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  const text = 'test';
  test('BarText render works correctly', () => {
    render(<BarText text={text} />);
  });
});
