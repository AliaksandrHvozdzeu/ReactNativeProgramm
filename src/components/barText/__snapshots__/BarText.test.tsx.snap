// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BarText tests Renders the Footer correctly 1`] = `
<Text
  style={
    {
      "color": "#F7F7F7",
      "fontFamily": "Roboto",
      "fontSize": 15,
      "fontStyle": "normal",
      "fontWeight": "500",
      "letterSpacing": 0.15,
      "lineHeight": 24,
      "textAlign": "center",
      "top": -10,
    }
  }
>
  test
</Text>
`;
